providers/provider.go

```diff
@@ -52,20 +58,34 @@ type SnapshotInfo struct {
 	Tags       map[string]string
 }
 
-type InstanceParameters struct {
-	MaintenanceWindow string `json:"maintenance_window"`
-	DailyBackupWindow string `json:"daily_backup_window"`
+type CacheParameter struct {
+	ParameterName  string `json:"parameter_name"`
+	ParameterValue string `json:"parameter_value"`
+}
+type ServiceParameters struct {
```

Comment:
The problem with renaming this to `ServiceParameters` is OSB lingo and specifically the `brokerapi` stubs use `Service` to mean the catalog entry (the parent object of the plans). The things we deal with really are "instances" in OSB lingo or more even "service instances" and mixing the two terms continues the confusion.

Reply:
Valid. We even refer to it as `instanceParameters` later.

Fixed.